src/EnvelopeCreator/DefineEnvelope.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DocusignBundle\EnvelopeCreator;
+
+use DocuSign\eSign\Model;
+use DocusignBundle\EnvelopeBuilder;
+use Symfony\Component\Routing\Router;
+use Symfony\Component\Routing\RouterInterface;
+
+class DefineEnvelope
+{
+    private $router;
+
+    public function __construct(RouterInterface $router)
+    {
+        $this->router = $router;
+    }
+
+    public function handle(EnvelopeBuilder $envelopeBuilder): void
```

Comment:
Class should be invokable + add an interface for autowiring, injection & testing